wormhole_attester/client/src/util.rs

```diff
@@ -179,3 +196,71 @@ pub async fn start_metrics_server(addr: impl Into<SocketAddr> + 'static) {
         .bind(addr)
         .await;
 }
+
+/// WARNING: Copied verbatim from v1.10.31, be careful when bumping
+/// solana crate versions!
+///
+/// TODO(2023-03-02): Use an upstream method when
+/// it's available.
+///
+/// This method is almost identical to
+/// RpcClient::send_and_confirm_transaction(). The only difference is
+/// that we let the user specify the config and replace
+/// send_transaction() inside with
+/// send_transaction_with_config(). This variant is currently missing
+/// from solana_client.
+pub async fn send_and_confirm_transaction_with_config(
```

Comment:
I think something we might consider is dropping the confirmation as well

Reply:
Haven't thought very deeply but are we getting anything from confirmation

Reply:
Yeah it makes it better but will make us blind on how much we are attesting successfully.

Reply:
Yeah, I think these results are critical to monitoring the service